src/IAM/Infrastructure/Persistence/MySQL/Migrations/Version20251217102810.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Twitter\IAM\Infrastructure\Persistence\MySQL\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20251217102810 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Create users table';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->addSql(
+            <<<SQL
+            CREATE TABLE users (
+                roles JSON NOT NULL,
+                email VARCHAR(320) NOT NULL,
+                password_hash VARCHAR(60) NOT NULL,
```

Comment:
```suggestion
                password_hash CHAR(60) NOT NULL,
```
@adnlv (medium) Please also check the mapping, so the diff won't change the structure.